src/transformers/models/led/modeling_led.py

```diff
@@ -1424,7 +1424,7 @@ class LEDSeq2SeqQuestionAnsweringModelOutput(ModelOutput):
         decoder_input_ids (:obj:`torch.LongTensor` of shape :obj:`(batch_size, target_sequence_length)`, `optional`):
             Provide for translation and summarization training. By default, the model will create this tensor by
             shifting the :obj:`input_ids` to the right, following the paper.
-        decoder_attention_mask (:obj:`torch.LongTensor` of shape :obj:`(batch_size, tgt_seq_len)`, `optional`):
+        decoder_attention_mask (:obj:`torch.LongTensor` of shape :obj:`(batch_size, target_sequence_length_length)`, `optional`):
```

Comment:
Woops, `length` ends up being here twice. Sorry about that!

Reply:
Thanks for re-checking!